site/main.css

```diff
@@ -124,33 +244,57 @@ th, td
 
 .centered { text-align: center; }
 
-code, kbd { font-size: 110%; }
+code, kbd, samp { font-size: 1.8rem; }
```

Comment:
I would agree, but this was making it 10% bigger. It's even more "subjective" because it relies on the size of the user's monospace font. I would probably go with removing the whole line then and not touch the `font-size`, allowing it to be inherited from the parent element.

Reply:
Good point. I wonder if there's a good solution to make them visually the same size. Maybe not touching the size is a sensible thing to do indeed.